### PR TITLE
Add documentation for HTTP Request & SDv4 verification plugin

### PR DESCRIPTION
**Pull request by shashankrnr32**
# Summary

@yahoo/ychaos-dev

#46

Documenting Verification Plugins

---

**Fixes**: #{issue_number}

#### Prerequisites
- [x] I have read the contribution guidelines

#### Code Analysis
- [ ] Covered by Unittests
- [ ] Security warnings ignored (Why?!)

#### Project related
- [ ] Schema changed and is backward compatible.
- [ ] Introduces a new sub-command for ychaos CLI

### Autogenerated Files
- [ ] Auto generated schema

### Pre-Merge Checklist

- [ ] All the build jobs are passing

---

### Checklist (Reviewer 1)

- [ ] Reviewed Source Code
- [ ] Reviewed Tests (If applicable)
- [ ] Reviewed Documentation (If applicable)

### Checklist (Reviewer 2)

- [ ] Reviewed Source Code
- [ ] Reviewed Tests (If applicable)
- [ ] Reviewed Documentation (If applicable)

### CODE REVIEW

**Review by sushilkar — APPROVED** [review]

File: docs/verification/plugins/http_request.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# HTTP Request Verification Plugin
+
+The HTTP Request Verification plugin is defined to make request calls to a particular endpoint
+and verify that the response from the server reaches the controller within an expected latency.
+This can be used to measure the latency of the server before, during and post the attack to montior
+the behaviour of your service during the phases of chaos testing.
+
+The plugin provides a number of configurations like Auth, Certificate, Headers etc. To view the schema of the
+configurations available for the plugin, visit [Verification Plugin][ychaos.testplan.verification.HTTPRequestVerification] in
```
Comment (AlfinST) [inline docs/verification/plugins/http_request.md:9]:
```suggestion
configurations available for the plugin, visit [Verification Plugin](ychaos.testplan.verification.HTTPRequestVerification) in
```

Comment (AlfinST) [reply docs/verification/plugins/http_request.md:9]: I was'nt aware of reference-style links using 
[reference-style link][Arbitrary case-insensitive reference text]

**Review by AlfinST — APPROVED** [review]
👍

File: docs/verification/plugins/http_request.md (hunk 1 above)
Comment (shashankrnr32) [reply docs/verification/plugins/http_request.md:9]: This is from mkdocstrings. Not typical Markdown syntax

**Pull request merged**